modules/twinkleblock.js

```diff
@@ -516,6 +516,16 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 		}
 	}
 
+	// grab discretionary sanctions list from en-wiki
+	var parent = this;
+	new Morebits.wiki.getCachedJson('Template:Ds/topics.json', parent).then(function(dsinfo) {
```

Comment:
`new` is not needed as getCachedJson is a simple function, not a class. The `parent` parameter can also be removed as it's not used anywhere. 

Sidenote: If we wanted to access `this`, it could be accessed anyway using `function(dsinfo) { ... }.bind(this)` so that `this` inside the function refers to the same thing as `this` outside the function.